language-support/ts/daml-ledger/README.md

```diff
@@ -6,7 +6,7 @@
 ## Documentation
 
 Comprehensive documentation for `@daml/ledger` can be found
-[here](https://docs.daml.com/app-dev/bindings-ts/daml-ledger/index.html).
+[here](https://docs.daml.com/0.0.0-SDKVERSION/app-dev/bindings-ts/daml-ledger/index.html).
```

Comment:
Does that actually work? @garyverhaegen-da

Reply:
I tested this locally with HEAD. Here it's replaces with '0.0.0'. This replace the version the same way in `packages.json`.

Reply:
I wasn't aware we run the substitution on the readmes as well. Good to know. Thanks for the explanation.